mkosi.profiles/obs/opensuse.repo

```diff
@@ -0,0 +1,7 @@
+[Base_System]
+name=Base:System Factory Devel Project (openSUSE_Tumbleweed)
+type=rpm-md
+baseurl=https://download.opensuse.org/repositories/system:/systemd/openSUSE_Tumbleweed/
+gpgcheck=0
```

Comment:
This should be `=1` instead of `=0`.

Reply:
This script also run too late, see https://github.com/systemd/mkosi/blob/main/mkosi/resources/man/mkosi.1.md#execution-flow there aren't any scripts called before the step2: "Sync the package manager repository metadata"

Reply:
@val4oss I'm disappointed something like this is being proposed without explicitly calling it out in the PR description. This is a conscious downgrade in security compared to other distros and not calling it out explicitly feels very lazy to me, especially coming from a suse employee who has way more influence in getting this fixed properly in zypper compared to myself.

https://github.com/systemd/mkosi/pull/4104 will generalize the workaround in mkosi so we import any keys listed in any repos in the rpmdb.

So please change this to `gpgcheck=1` as @Conan-Kudo suggested. And then copy the gpg key to /usr/share/pki/rpm-gpg in the sandbox tree like is done for Fedora and list it as the gpgkey in the repo file here, same as Fedora.